pkgs/data/icons/nordzy-icon-theme/default.nix

```
@@ -0,0 +1,56 @@
+{ stdenvNoCC
+, fetchFromGitHub
+, lib
+, gtk3
+, nordzy-theme-name ? "Nordzy"
+, nordzy-themes ? [ "all" ] # Override this to only install selected themes
+}:
+
+let
+  themes-arg-string = lib.strings.concatMapStrings (theme: "-t ${theme} ") nordzy-themes;
+in
+stdenvNoCC.mkDerivation {
+  pname = "nordzy-icon-theme";
+  version = "unstable-2021-12-14";
+
+  src = fetchFromGitHub {
+    owner = "alvatip";
+    repo = "Nordzy-icon";
+    rev = "5c247a4f19cf9849615631d1bf77727b945b634e";
+    sha256 = "Jqn5CF80xlYJ7H4qI1VEj91vcKPPoMXP5+sPs0ksiC4=";
```

```suggestion
    rev = "ee5d01733fb69c05f5f4531105036148c47f9419";
    sha256 = "HRBPBZ3Q3ZHRcGcI1zxOI2PWO9/dkkOu9JdOdpIQxac=";
```
Use the latest revision.